### PR TITLE
Create twilio@2.36.1.rb

### PR DESCRIPTION
**Pull request by shamantraghav**
Create twilio@2.36.1.rb

Documentation:  https://docs.brew.sh/Versions

Older release: https://github.com/twilio/homebrew-brew/commit/a4b0c0306b84a22b3be6c8738ed857c4b7a8336b

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: Formula/twilio@2.36.1.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class TwilioAT2361 < Formula
+  desc "unleash the power of Twilio from your command prompt"
+  homepage "https://github.com/twilio/twilio-cli"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.36.0/twilio-v2.36.0.tar.gz"
+  version "2.36.0"
+  sha256 "d896744ab6518e0aaab4cb0ce069a2c9185b9732716b0cd4de0725bb011c5194"
```
Comment (onuzbee) [inline Formula/twilio@2.36.1.rb:8]: mismatch in `version` and `sha256`

Title says version `2.36.1` but this'd install 2.36.0

Comment (onuzbee) [reply Formula/twilio@2.36.1.rb:8]:
```
  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.36.1/twilio-v2.36.1.tar.gz"
  version "2.36.1"
  sha256 "9c57ba33583ebf7c3c2ca20100554a9c2050c6a707a51bdae71fac18e64c0e21"

```

Comment (shrutiburman) [reply Formula/twilio@2.36.1.rb:8]: +1, [commit](https://github.com/twilio/homebrew-brew/commit/3a5bc4aabe0fc1837a2b36a08894c5490715c910)

Comment (shamantraghav) [reply Formula/twilio@2.36.1.rb]: Fixed this, thanks

**Review by kridai — APPROVED** [review]

**Review by shrutiburman — APPROVED** [review]

**Review by onuzbee — APPROVED** [review]

**Pull request merged**